### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by machacekondra**
Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: manifests/vm-import-operator/v0.2.0/operator.yaml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  template:
+    metadata:
+      labels:
+        name: vm-import-operator
+    spec:
+      serviceAccountName: vm-import-operator
+      containers:
+        - name: vm-import-operator
+          # Replace this with the built image name
+          image: quay.io/omachace/vm-import-operator:latest
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/operator.yaml]: Please replace with quay.io/kubevirt/vm-import-operator:v0.2.0

File: manifests/vm-import-operator/v0.2.0/operator.yaml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "vm-import-operator"
+            - name: DEPLOY_CLUSTER_RESOURCES
+              value: "true"
+            - name: OPERATOR_VERSION
+              value: latest
+            - name: CONTROLLER_IMAGE
+              value: quay.io/omachace/vm-import-controller:latest
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/operator.yaml]: Please replace with quay.io/kubevirt/vm-import-controller:v0.2.0

File: manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                operator.v2v.kubevirt.io: ""
+            spec:
+              containers:
+              - env:
+                - name: DEPLOY_CLUSTER_RESOURCES
+                  value: "true"
+                - name: OPERATOR_VERSION
+                  value: latest
+                - name: CONTROLLER_IMAGE
+                  value: quay.io/omachace/vm-import-controller:latest
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml]: Please replace with quay.io/kubevirt/vm-import-controller:v0.2.0

File: manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+                - name: PULL_POLICY
+                  value: Always
+                - name: WATCH_NAMESPACE
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                image: quay.io/omachace/vm-import-operator:latest
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml]: Please replace with quay.io/kubevirt/vm-import-operator:v0.2.0

**Review by jakub-dzon — CHANGES_REQUESTED** [review]

File: manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml (hunk 4 above)
Comment (machacekondra) [reply manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml]: Thanks, for finding it I wonder, why the setup script didn't replace it...

**Review by pkliczewski — APPROVED** [review]

File: manifests/vm-import-operator/v0.2.0/operator.yaml [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "vm-import-operator"
+            - name: DEPLOY_CLUSTER_RESOURCES
+              value: "true"
+            - name: OPERATOR_VERSION
+              value: latest
+            - name: CONTROLLER_IMAGE
+              value: quay.io/kubevirt/vm-import-operator:v0.2.0
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/operator.yaml]: controller

File: manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+                operator.v2v.kubevirt.io: ""
+            spec:
+              containers:
+              - env:
+                - name: DEPLOY_CLUSTER_RESOURCES
+                  value: "true"
+                - name: OPERATOR_VERSION
+                  value: latest
+                - name: CONTROLLER_IMAGE
+                  value: quay.io/kubevirt/vm-import-operator:v0.2.0
```
Comment (jakub-dzon) [inline manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml]: controller

**Review by jakub-dzon — CHANGES_REQUESTED** [review]

File: manifests/vm-import-operator/v0.2.0/operator.yaml (hunk 5 above)
Comment (pkliczewski) [reply manifests/vm-import-operator/v0.2.0/operator.yaml]: Good catch!

File: manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml (hunk 6 above)
Comment (pkliczewski) [reply manifests/vm-import-operator/v0.2.0/vm-import-operator.0.2.0.clusterserviceversion.yaml]: same here

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**